Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:6.9
+
+RUN apt-get update && apt-get -qq install -y postgresql-client curl gzip
+
+WORKDIR /app
+ADD package.json /app/package.json
+RUN npm install
+
+ADD server.js /app/server.js
+ADD config.js /app/config.js
+ADD controllers /app/controllers
+ADD services /app/services
+ADD ddl /app/ddl
+ADD docker-entrypoint.sh /app/docker-entrypoint.sh
+
+CMD /app/docker-entrypoint.sh
+
+ENV PORT=80
+EXPOSE 80
```

Comment:
Do you know if it's possible to do a `EXPOSE $PORT`?

Reply:
Not sure. From the [docs/spec](https://docs.docker.com/engine/reference/builder/#expose) it seems like only constants can be exposed. I'll give it a try though and report back.

Reply:
Yes, seems to work. Good catch. See 453cccc